src/Components/PetList.js

```diff
@@ -1,9 +1,17 @@
-export const PetList = () => {
-  return (
-    <aside className="pets-list">
-      <p>No pets listed for this employee.</p>
-    </aside>
-  );
-};
+import { useState, useEffect } from "react";
 
-export default PetList;
+export const PetList = () => {
+  const [pets, setPets] = useState([]);
+  useEffect(() => {
+    fetch(`https://vet-app-0obi.onrender.com/api/pets`)
```

Comment:
You have the wrong URL here. You'd wanna use `/api/pets?employeeId=vlJtFOU` so you would need to pass the employee `id` as a prop